NOTICE.txt

```diff
@@ -8,7 +8,7 @@ This document includes a list of open source components used in Mattermost Mobil
 
 --------
 
-This product contains a modified portion of 'fbjs', a collection of JavaScript utilities by Facebook, Inc.
+This product contains 'fbjs', a collection of JavaScript utilities by Facebook, Inc.
```

Comment:
This one should probably stay "a modified portion" since that one had to be modified slightly to pass our style guidelines because of how it's included